src/vs/editor/contrib/stickyScroll/browser/stickyScrollController.ts

```diff
@@ -321,6 +330,26 @@ export class StickyScrollController extends Disposable implements IEditorContrib
 		}
 	}
 
+	private _readAccessibilitySupportConfiguration() {
+		const accessibilitySupport = this._editor.getOption(EditorOption.accessibilitySupport);
+		// Accessibility support has been enabled
+		if (accessibilitySupport === AccessibilitySupport.Enabled) {
+			const stickyScrollEnabled = this._editor.getOption(EditorOption.stickyScroll).enabled;
+			this._storageService.store(StickyScrollController._key, stickyScrollEnabled, StorageScope.PROFILE, StorageTarget.USER);
+			if (!stickyScrollEnabled) {
+				this._configurationService.updateValue('editor.stickyScroll.enabled', true);
+			}
+		}
+		// Accessibility support has been disabled
+		else {
+			const currentStickyScrollEnabled = this._editor.getOption(EditorOption.stickyScroll).enabled;
+			const storedStickyScrollEnabled = !!JSON.parse(this._storageService.get(StickyScrollController._key, StorageScope.PROFILE) ?? `false`);
+			if (currentStickyScrollEnabled !== storedStickyScrollEnabled) {
+				this._configurationService.updateValue('editor.stickyScroll.enabled', storedStickyScrollEnabled);
```

Comment:
I have generally mixed feelings about different default configs based on the accessibility mode being enabled. I think this is mere a personal preference and not so much about accessibility. If you ask me we should enable sticky scroll for everyone 😉

Anyways, this implementation isn't very polite because it changes a setting. If we do something special we for accessiblity mode than it should be a "dynamic default", but not something that gets persisted. You would need to check if stickyScroll is unconfigured (which is different than configured to default value) and only than use the accessibility mode.